contracts/adapters/bzx/BzxStakingAdapter.sol

```diff
@@ -0,0 +1,104 @@
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import {ERC20} from "../../ERC20.sol";
+import {ProtocolAdapter} from "../ProtocolAdapter.sol";
+
+/**
+ * @dev StakingRewards contract interface.
+ * Only the functions required for AragonStakingAdapter contract are added.
+ * The StakingRewards contract is available here
+ * github.com/Synthetixio/synthetix/blob/master/contracts/StakingRewards.sol.
+ */
+interface StakingRewards {
+
+    function balanceOfByAssets(address account)
+        external
+        view
+        returns (
+            uint256 bzrxBalance,
+            uint256 iBZRXBalance,
+            uint256 vBZRXBalance,
+            uint256 lPTokenBalance
+        );
+
+    function balanceOfByAsset(address token, address account)
+        external
+        view
+        returns (uint256 balance);
+}
+
+/**
+ * @title Adapter for BZX protocol (staking).
+ * @dev Implementation of ProtocolAdapter interface.
+ * @author Roman Iftodi <romeo8881@gmail.com>
+ */
+contract BzxStakingAdapter is ProtocolAdapter {
+    string public constant override adapterType = "Asset";
+
+    string public constant override tokenType = "ERC20";
+
+    address internal constant IBZRX =
+        0x18240BD9C07fA6156Ce3F3f61921cC82b2619157;
+    address internal constant VBZRX =
+        0xB72B31907C1C95F3650b64b2469e08EdACeE5e8F;
+    address internal constant BZRX = 0x56d811088235F11C8920698a204A5010a788f4b3;
+    address internal constant BPT = 0xe26A220a341EAca116bDa64cF9D5638A935ae629;
+
+    address internal constant stakingContract =
+        0xe95Ebce2B02Ee07dEF5Ed6B53289801F7Fc137A4;
+
+    /**
+     * @return Amount of staked LP tokens for a given account.
+     * @dev Implementation of ProtocolAdapter interface function.
+     */
+    function getBalance(address token, address account)
+        external
+        view
+        override
+        returns (uint256)
+    {
+        if (token == IBZRX) {
+            return
+                StakingRewards(stakingContract).balanceOfByAsset(
+                    token,
+                    account
+                );
+        } else if (token == VBZRX) {
+            return
+                StakingRewards(stakingContract).balanceOfByAsset(
+                    token,
+                    account
+                );
+        } else if (token == BZRX) {
+            return
+                StakingRewards(stakingContract).balanceOfByAsset(
+                    token,
+                    account
+                );
+        } else if (token == BPT) {
+            return
+                StakingRewards(stakingContract).balanceOfByAsset(
+                    token,
+                    account
+                );
```

Comment:
all 3 cases can be combined into one if-case (`if (token == IBZRX || token == VBZRX || ... )`)